File: test/fixtures/elixir/get_basic_auth_no_user.ex
request = %HTTPoison.Request{
  method: :get,
  url: "http://localhost:28139/",
  options: [hackney: [basic_auth: {"", ~s|some_password|}]],
  headers: [],
  params: [],
  body: ""
}

response = HTTPoison.request(request)

File: test/fixtures/kotlin/get_basic_auth_no_user.kt
import java.io.IOException
import okhttp3.Credentials
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val credential = Credentials.basic("", "some_password");

val request = Request.Builder()
  .url("http://localhost:28139/")
  .header("Authorization", credential)
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}

